tests/lib/bash/test_header

```diff
@@ -191,6 +194,17 @@ run_fail() {
     ok "${TEST_NAME}"
 }
 
+run_graphql_ok () {
+    local TEST_NAME="$1"
+    local SUITE="$2"
+    local QUERY="$3"
+    # shellcheck disable=SC2086
+    run_ok "${TEST_NAME}" cylc client \
+        "${SUITE}" \
+        'graphql' \
+        <<< "{\"request_string\": "\"${QUERY}\""}"
```

Comment:
I can't seem to run the test successfully;
`$ ./etc/bin/run-functional-tests.sh -v ./tests/graphql/01-workflow.t`
on my system.

It works if I change the `here` redirect to:
`. . . < <(echo ${REQ_STRING})` or `echo ${REQ_STRING} | . . .`
That is:
```
#!/bin/bash

func1 () {
    "$@"
}

func2 () {
    local REQ_STRING=$1
    echo <<< ${REQ_STRING} 
    func1 cylc client baz graphql < <(echo ${REQ_STRING})
    #echo ${REQ_STRING} | func1 cylc client baz graphql
}

func2 '
{"request_string": "
  query {
      workflows {
          name,
          status,
          statusMsg
      }
  }",
"variables": null}
'
```

Reply:
This is nicer!

Reply:
Don't get me wrong I do like `<<<`, it's only that I can't get;
```$ ./etc/bin/run-functional-tests.sh -v ./tests/graphql/01-workflow.t```
working with it on my Ubuntu VM   (>.<)

Reply:
Pushed, hopefully this will work for you now?

Reply:
Ya, good.